packages/frontend-2/lib/viewer/composables/setup.ts

```diff
@@ -75,6 +77,11 @@ export type LoadedThreadsMetadata = NonNullable<
 
 export type LoadedCommentThread = NonNullable<Get<LoadedThreadsMetadata, 'items[0]'>>
 
+interface MeasurementState {
+  enabled: boolean
```

Comment:
Ideally these 2 properties would be 2 different refs, cause now a change to `options` will mark the entire state as updated and possibly trigger code that's only interested in updates to `enabled`.